ref_builder/ncbi/client.py

```diff
@@ -40,6 +43,26 @@ class GenbankRecordKey(StrEnum):
     SEQUENCE = "GBSeq_sequence"
 
 
+@dataclass
+class NuccoreSearchTerms:
+    sequence_min_length: int = 0
+
+    sequence_max_length: int = MAX_SEQUENCE_LENGTH
```

Comment:
Get rid of `MAX_SEQUENCE_LENGTH` and use a default of `0` which means no max is applied.